clang/test/CIR/CodeGen/agg-init.cpp

```diff
@@ -1,8 +1,8 @@
 // RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -std=c++17 -fclangir-enable -Wno-unused-value -emit-cir %s -o %t.cir
 // RUN: FileCheck --input-file=%t.cir %s
 
-// CHECK: !ty_22struct2EZero22 = !cir.struct<"struct.Zero" {!u8i}>
-// CHECK: !ty_22struct2Eyep_22 = !cir.struct<"struct.yep_" {!u32i, !u32i}>
+// CHECK: !ty_22struct2EZero22 = !cir.struct<struct "struct.Zero" {!u8i}>
```

Comment:
While here, can you please take the opportunity to drop all `"struct." and "class."` string prefixes from the names? Now that we have the actual name as a keyword it doesn't make sense to keep the string prefix.